partners/anaconda/base/anaconda-ce-validator-cron.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: anaconda-ce-periodic-validator
+  namespace: redhat-ods-applications
+  labels:
+    opendatahub.io/modified: "false"
+spec:
+  schedule: "0 0 * * *"
```

Comment:
This looks like it checks once a day, is that enough?

Reply:
This is only the logic to disable it in the UI, if the token expires, it's useless to the user as far as using it for Anaconda stuff, I figured this was a good compromise between resource usage and checking often enough